### ISSUE TITLE
Disable widescreen support for cutscenes only option?

### ISSUE THREAD

**Issue by ZenithWest** [Lyall/FFXVIFix#87]
Not sure if this is possible, but I'm on a 32:9 aspect ratio HDR monitor.... During certain cutscenes where the dev didn't actually render anything leaving a bright white background, the left and right sides of my screen become blindingly bright (and of course its during a very dark night cutscene). Its like having two spot-lights shining in my face. Ideally I wish there was a way just to have those specific scene's edges blacked out, but it really ruins the cutscene when it happens.

**Comment by TheFlandy** [comment]
Yeah it'd be really nice if there was an option for black bars during cutscenes
